### DISCUSSION TITLE
OCPP installed with HACS but does not show up in Devices&Services - Add Integration

### DISCUSSION THREAD

**Discussion by BirgerKo** [lbbrhzn/ocpp#504] [Q&A]
Hi,

I would really appreciate som help here. 
I'm trying to get OCPP integration in to use on a Wallbox Pulsar Pluss.
I already have the Wallbox integration running, but I would like to use a OCPP solution on one of the charges to learn more.

**System info:**
Version | core-2022.4.7
Installation Type | Home Assistant OS
Development | false
Supervisor | true
Docker | true
User | root
Virtual Environment | false
Python Version | 3.9.9
Operating System Family | Linux
Operating System Version | 5.10.103-v8
CPU Architecture | aarch64
Timezone | Europe/Oslo

Installed OCPP through HACS
OCPP version v0.4.31
Restarted Home Assistant
Try to find OCPP in Configuration/Devices&Services/Add Integration
Nothing shows up so I never get to the config page.

I'm running with this logging:
logger:
  default: info
  logs:
    custom_components.ocpp: debug
  
Only one line in the log mentions OCPP:
2022-04-27 18:58:12 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration OCPP which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant

Is there anything more I can look at?
I have removed and reinstalled several times.

Kind regards / Birger

**Comment by ayryq** [comment]
For me this just worked. Here's my process just to make sure:
```
HACS - explore and download repositories - search for and select ocpp - click "download this repo..." - download - should show "pending restart" in HACS - restart HA - no need to return to HACS now

Configuration/Integrations - Add Integration - search ocpp - ocpp integration should appear in list
```
So probably you've done all that and it's still not working. HACS can be fussy with browser cache; have you tried clearing the cache, or loading in a different browser than usual? This is a not-uncommon problem for HACS integrations, so you might find more help if you don't specify OCPP when searching. Some other threads I just read suggest rebooting the hardware (not just restarting HA), or logging your user out and and back in.

**Reply by Flix6x** [reply]
@ayryq Thank you! I ran into the same problem, and reloading in a different browser indeed made the integration appear in the list.

**Comment by BirgerKo** [comment]
Thanks for the answer.
I did not have time to look at it for awhile. Yesterday I did and first I
upgrade home assistant and it worked after that.
